src/vc/vc.ts

```diff
@@ -457,7 +457,7 @@ export class Vc extends Logger {
       throw Error(`Invalid issuer DID: ${vc.issuer.id}`);
     }
 
-    if (this.options.signerIdentity.activeIdentity.toLowerCase() !== issuerIdentity) {
+    if (this.options.signerIdentity.activeIdentity.toLowerCase() !== issuerIdentity.toLowerCase()) {
```

Comment:
```suggestion
    if (this.options.signerIdentity.activeIdentity !== issuerIdentity) {
```
Can't we just compare both identities as they are?
- `this.options.signerIdentity.activeIdentity` should be checksum case already
- `convertDidToIdentity` now calls `this.options.web3.utils.toChecksumAddress` when length is 42